### DISCUSSION TITLE
Change tool to use Litematica

### DISCUSSION THREAD

**Discussion by Beppe0515** [maruohon/litematica#711] [Q&A]
Hi, I would want to change the item to use litematica. Currently by default its a stick but i am not able to change it to like a heart of the sea (an item i wouldn't use otherwise). Everytime i write something into the box provided (currently it says minecraft:stick), it doesnt save what i wrote, whenever i exit the menu. What do i do to change the item to use litematica? 

Ps. I have seen a youtuber named Wunba do it to a netherite chestplate.

**Comment by maruohon** [comment]
You need to make sure you are using the correct registry name of the item. If the item can't be found, it reverts the value back to the default stick. So in your case that would be `minecraft:heart_of_the_sea`.

But in general I would recommend using any item that you always have on you, not an extra item you don't use for anything else. You can also use an empty hand as the tool by leaving the text field empty. And then just turn on/off the tool item functionality as needed with `M + T` or Generic -> `toolItemEnabled`.

In my case I use either my usual food item as the tool, or my pickaxe. There is no point wasting an inventory slot for an extra item.

**Reply by Beppe0515** [reply]
Ok thanks. What i thought was you would write it minecraft:heart:of:the:sea or minecraft_heart_of_the_sea. I didnt know it was supposed to be like minecraft:heart_of_the_sea. Also thanks for the advice by using a common used item. I didnt want to because it would be very annoying when accidently clicking while holding that item. Now i know you can enable/disable it.

**Reply by maruohon** [reply]
If you enable the advanced item tooltips with `F3 + H`, then you can see the registry name of the item in the tooltips, like here:
![image](https://github.com/maruohon/litematica/assets/2331050/863381ab-afa2-44d2-ac6b-e40e58a0ffae)

Also note that in the current mod versions in 1.13+ the tool item check is strict and it won't allow extra NBT data on the items. So items with enchantments or durability won't work as tool items. This will be fixed once the newer code gets ported from 1.12.2 at some point, there the check was relaxed such that it ignores the NBT data otherwise, but if you specify some NBT tags in the tool item text field, then those tags are required to exist in the item. This was added because someone wanted to use specially named items as the tool. But like I explained above, in my opinion that is pointless and just wastes inventory space for pointless extra items. IMO it's better to just use whatever you usually carry on you anyway, and then only enable the tool item functionality while you actually need to use it.

**Reply by Beppe0515** [reply]
OK thank you
